docs/Monitoring.md

```diff
@@ -30,8 +30,7 @@ kamon {
     sampler = "random"
   }
 
-}
-```
+}```
```

Comment:
Why did you change that? Just look at the result on github and see that it breaks the formatting: https://github.com/GoutamVerma/eclair/blob/prometheus_monitoring/docs/Monitoring.md

Please revert that and fix it line 71 as well.

Reply:
Now it's looks better
https://github.com/GoutamVerma/eclair/blob/prometheus_monitoring/docs/Monitoring.md